tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = typing,pylint,docs,twinecheck,pre-commit,py38,py39,py310,py311,py312,pypy3.9,pypy3.10
+envlist = typing,pylint,docs,twinecheck,pre-commit,py39,py310,py311,py312,py313,pypy3.10
```

Comment:
I wonder if, instead of removing PyPy 3.9 from here, we should add it to GitHub Actions.